setup.py

```diff
@@ -59,36 +59,13 @@
 for name, reqs in extras_require.items():
     extras_require['all'].extend(reqs)
 
-URL = 'https://github.com/inspirehep/inspire-mitmproxy'
-
 setup(
-    name='inspire-mitmproxy',
-    author="CERN",
-    author_email='admin@inspirehep.net',
-    autosemver={
-        'bugtracker_url': URL + '/issues',
-    },
     description=__doc__,
     long_description=readme,
-    url=URL,
-    license='GPLv3',
-    zip_safe=False,
-    include_package_data=True,
-    platforms='any',
+    url='https://github.com/inspirehep/inspire-mitmproxy',
     install_requires=install_requires,
     tests_require=tests_require,
     setup_requires=setup_requires,
     extras_require=extras_require,
```

Comment:
I believe that with the current configuration the test folder will be included in the artifact created. We could improve this config to exclude it.

Reply:
As discussed IRL, tests are not actually added to the package